galley/pkg/runtime/processor.go

```diff
@@ -114,9 +112,39 @@ func newProcessor(
 	}))
 
 	p.handler = buildDispatcher(p.state, p.serviceEntryHandler)
+
+	p.seedMesh()
+
 	return p
 }
 
+func (p *Processor) seedMesh() {
+	i, ok := p.state.config.Schema.Lookup("istio/mesh/v1alpha1/MeshConfig")
+	if !ok {
+		return
+	}
+	m := p.state.config.Mesh.Get()
+	e := resource.Event{
+		Kind: resource.Added,
+		Entry: resource.Entry{
+			Metadata: resource.Metadata{
+				CreateTime: time.Now(),
+			},
+			ID: resource.VersionedKey{
+				Key: resource.Key{
+					Collection: i.Collection,
+					FullName: resource.FullNameFromNamespaceAndName(
+						"istio-system",
```

Comment:
Can you add a TODO to replace istio-system with the appropriate name? We may need to serve multiple meshconfigs, and this name can cause issues.